### PR TITLE
docs: (Core) Added missing cards examples

### PR DESCRIPTION
**Pull request by DeepakSap14**
#### Please provide a link to the associated issue.
#3863 #3864 #3866 #3869 #3870 #4243 

#### Please provide a brief summary of this pull request.
1. Added cards for the raised issue.
2. Have only list variant for "link list card" #3864 . Image variant is pending.

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

File: apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<fd-card cardType="object" class="docs-flex-item-margin">
+    <fd-card-header>
```
Comment (JKMarkowski) [inline apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.html:2]: Could you add `[interactive]="false"` there? card with link list shouldn't have hoverable/focusable header

File: apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<fd-card cardType="object" class="docs-flex-item-margin">
+    <fd-card-header>
```
Comment (JKMarkowski) [inline apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.html:2]: Could you add `[interactive]="false"` there? card with list shouldn't have hoverable/focusable header

**Review by JKMarkowski — COMMENTED** [review]
Hello @DeepakSap14 LGTM, but I added 2 comments regarding headers. header `[interactive]` property is already on main branch

File: apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.html (hunk 1 above)
Comment (DeepakSap14) [reply apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.html]: fixed.

File: apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.html (hunk 2 above)
Comment (DeepakSap14) [reply apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.html]: fixed.

**Comment by puru-hk** [comment]
#3863 List card is not aligned properly attache screenshot
image

**Comment by puru-hk** [comment]
#4243 KPI color should be implemented.
Screenshot
image

File: apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-card-link-list-example',
+    templateUrl: 'card-link-list-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.ts:6]: do you need this?

File: apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.scss [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+    color: red;
+}
+
+.bg-ok {
+    color: orange;
+}
+
+.bg-positive {
+    color: green;
+}
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.scss:11]: Please use appropriate naming. It should also contain the word `docs` so the user is not confused where this class is coming from

File: apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+@Component({
+    selector: 'fd-card-list-example',
+    templateUrl: 'card-list-example.component.html',
+    styleUrls: ['./card-list-example.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush
```
Comment (InnaAtanasova) [inline apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.ts:7]: I don't think you need OnPush in this case. You have no component logic here.

**Review by InnaAtanasova — COMMENTED** [review]
Changed the title to `docs` since you are only touching docs files. Make sure it also says `docs` when you are merging the PR

**Comment by DeepakSap14** [comment]
@puru-hk 
1. #3863 List card is not aligned properly attache screenshot
2. #4243 KPI color should be implemented.
These are corrected now. please check.

File: apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.ts (hunk 5 above)
Comment (DeepakSap14) [reply apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.ts]: removed OnPush ChangeDetectionStrategy

File: apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.scss (hunk 4 above)
Comment (DeepakSap14) [reply apps/docs/src/app/core/component-docs/card/examples/list-card/card-list-example.component.scss]: fixed.

File: apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.ts (hunk 3 above)
Comment (DeepakSap14) [reply apps/docs/src/app/core/component-docs/card/examples/link-list-card/card-link-list-example.component.ts]: removed OnPush ChangeDetectionStrategy. fixed

**Comment by puru-hk** [comment]
#3863 count number color is wrong: 
screen shot:
image

expected:
image

#3866 Please add one example with action button 

expected:
image

#3869 when it change to month and year it should change to "For This Month" and "For This Year"
expected: 
image

#4243 Looks fine

**Comment by puru-hk** [comment]
Looks fine

**Review by puru-hk — APPROVED** [review]

**Review by JKMarkowski — APPROVED** [review]
LGTM

**Pull request merged**